### PR TITLE
Rename usStackDepth and usStackHighWaterMark

### PR DESCRIPTION
**Pull request by cookpate**
Rename usStackDepth and usStackHighWaterMark

Description
-----------

Refer to https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/942
That PR also renames these variables. This PR would need to also be accepted in order to continue building demos and tests in CI.

Test Steps
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [x] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS/Demo/WIN32-MingW/code_coverage_additions.c [hunk 1]
```diff
@@ -94,39 +94,39 @@ UBaseType_t uxDummy = 10;
 	passed in place of a required buffer.  Hence if all passes then none of the
 	|= will be against 0, and ulReturned will still be zero at the end of this
 	function. */
-	ulReturned |= ( uint32_t ) xEventGroupCreateStatic( NULL );
+	ulReturned |= ( uint32_t ) ( uintptr_t ) xEventGroupCreateStatic( NULL );
```
Comment (cookpate) [inline FreeRTOS/Demo/WIN32-MingW/code_coverage_additions.c:97]: Oops, sneaking this change in. Suppresses a pointer-to-int conversion warning. Might be better to make ulReturned `uintptr_t`, here. Might revert

Comment (n9wxu) [reply FreeRTOS/Demo/WIN32-MingW/code_coverage_additions.c:97]: uintptr_t would be better.  Also single purpose PR's are easier to review.

**Review by n9wxu — APPROVED** [review]

**Review by sirnish — APPROVED** [review]
looks gtg

**Comment by kar-rahul-aws** [comment]
`TaskParameters_t.usStackDepth` and `TaskStatus_t.usStackHighWaterMark` are not updated in the PR# [942](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/942)
So this PR is no longer necessary, can we close it?

**Comment by aggarg** [comment]
Closing the PR as per @kar-rahul-aws's comment.

**Pull request closed without merging**